Branch-SDK/Branch-SDK/BNCServerInterface.h

```diff
@@ -18,6 +19,8 @@ static NSString *REQ_TAG_DEBUG_DISCONNECT = @"t_debug_disconnect";
 
 @interface BNCServerInterface : NSObject
 
+@property (strong, nonatomic) BNCPreferenceHelper *preferenceHelper;
```

Comment:
Not a big deal, just feel a bit inconsistent: in some cases, e.g. here, pref helper is a property, and used like [self.preferenceHelper ...]; in some other cases, it's used with the singleton syntax, e.g. [[BNCPeferenceHelper preferenceHelper] ...].